README.md

```diff
@@ -75,6 +75,8 @@ The following Plumber Functions are currently available:
 
 - `ABS(number)`: Returns the absolute value of a number
 - `POWER(base, exponent)`: Returns the result of a number raised to a power
+- `STR_REPLACE_ALL(input, search, replace)`: Replace all instances of a substring using a search string.
+- `UNIDECODE(string)`: Converts UTF-8 data into ASCII characters, mostly by transliteration (pronunciation).
```

Comment:
I'm thinking there's another potential approach we can take with `unidecode-plus` to address the potential risk I discussed.

Recap: potential risk its unclear how unidecode actually converts to ASCII - we may end up with surprises, such as ― (long dash `\u2015`) decoding to 2 dashes `--` instead of 1 dash `-`.

## Option 1
We allow users to minimize the risk by constraining the characters to convert. We can do this by allowing the caller to specify the range of characters they want to convert.

Let's consider the general punctuation unicode range: https://jrgraphix.net/r/Unicode/2000-206F

Proposal: we can provide a `UNIDECODE(string, code_point_start, code_point_end)` function with these args:
- `string`: String to process
- `code_point_start`: The 1st code point in decimal to start replacing (e.g. 8109 for 1st char in punctuation range - `0x2000`)
- `code_point_end`: The last code point in decimal to start replacing (e.g. 8303 for last char in punctuation range `0x206F`)

So calling  "UNIDECODE("読み―", 8109, 8303)" would only transform the last "―" character. The intent is to allow users to test what the output is first.

## Option 2
Stick with current option `UNIDECODE(string)`

---
Does this make sense? If so, which option do you think is better?

As an aside, I'm also thinking to code option 1 directly into Plumber's PaySG action - less complexity for the users.

Reply:
Actually, what if we could have the best of both worlds? We could offer a function `UNIDECODE(string, [code_point_start, code_point_end])` that allows for the last two arguments to be optional, that default to the punctuation range.

We just need to extend the language to allow for a flexible number of function arguments.